perun-cli/anonymizeUser

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+use Getopt::Long qw(:config no_ignore_case);
+use Perun::Agent;
+use Perun::Common qw(printMessage);
+
+sub help {
+	return qq{
+	Anonymizes a user. User id is required.
+	--------------------------------------
+	Available options:
+	--userId   | -u user id
+	--batch      | -b batch
+	--help       | -h prints this help
+
+	};
+}
+
+my ($userId, $batch);
+GetOptions ("help|h" => sub {
+		print help();
+		exit 0;
+	}, "batch|b"     => \$batch,
+	"userId|m=i"     => \$userId) || die help();
```

Comment:
I think the short version of the `userId` parameter should be `u` and not `m`

Reply:
fixed